Mathlib/CategoryTheory/Triangulated/HomologicalFunctor.lean

```diff
@@ -230,7 +230,7 @@ lemma mem_homologicalKernel_W_iff {X Y : C} (f : X ⟶ Y) :
   simp only [mem_homologicalKernel_iff, h₁, ← h₂, ← h₃]
   constructor
   · intro h n
-    obtain ⟨m, rfl⟩ : ∃ (m : ℤ), n = m + 1 := ⟨n - 1, by omega⟩
+    obtain ⟨m, rfl⟩ : ∃ (m : ℤ), n = m + 1 := ⟨n - 1, by simp⟩
```

Comment:
(Note: with `omega`, irrelevant unnecessary variables were added to the lemma. Using `simp` solves this issue.)